ros2_ws/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.5)
+project(pointcloud2_pubsub)
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+
+add_executable(publisher_node src/publisher_node.cpp)
+ament_target_dependencies(publisher_node rclcpp sensor_msgs)
+
+add_executable(subscriber_node src/subscriber_node.cpp)
+ament_target_dependencies(subscriber_node rclcpp sensor_msgs)
+
+install(TARGETS
+  publisher_node
+  subscriber_node
+  DESTINATION lib/${PROJECT_NAME}
```

Comment:
The installation destination `lib/${PROJECT_NAME}` is generally acceptable for ROS2 packages. However, for better organization within the installation, consider using a more descriptive destination path, such as `lib/${PROJECT_NAME}/` (adding a trailing slash). This ensures better separation if there are other files or directories to be installed along with the executables.